content/30-use-pah/_index.md

```diff
@@ -7,106 +7,131 @@ weight = 30
 
 ### Prerequisites
 
-* Automation Controller is running and accessible
+* Make sure you can login to your:
 
-* Private Automation Hub is running and accessible
+  * Automation Controller
+  * Private Automation Hub (PAH)
 
+* Using the access information/credentials provided
 ### Tasks
 
+* Add collections from Red Hat Automation Hub and Galaxy to PAH
 * Configure Automation Controller to access your Private Automation Hub
-
-* Add collections from Red Hat Automation Hub, Galaxy and custom ones created by you.
-
-* Add execution environment images
+* Use a collection from PAH in Automation Controller
 
 Let's start, as the docs for this are distributed over some places we'll give some more instructions.
 
-### Add content to your private Automation Hub
+### Add and use content in your Private Automation Hub
 
 Most of this is well documented [here](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/managing_red_hat_ansible_content_collections_and_ansible_galaxy_collections_in_automation_hub/index)
 
-#### Sync collections from Red Hat Automation Hub
+#### Sync collections from Red Hat Automation Hub to PAH
 
 * Go to `console.redhat.com` and open **Ansible Automation Platform** -> **Automation Hub** -> **Collections**. Here you can enable/disable the sync of certain collections.
```

Comment:
maybe we should mention you need "org admin" privileges - if you don't ask one of your friendly Red Hatters around you to give you his/her API token